app/models/user.rb

```diff
@@ -5,7 +5,40 @@ class User < ApplicationRecord
   include Searchable
   include Storext.model
 
-  self.ignored_columns = %w[currently_streaming_on twitch_username]
+  # @citizen428 Preparing to drop profile columns from the users table
+  PROFILE_COLUMNS = %w[
```

Comment:
Just noticed that `name` needs to be added to this list too because [it's being used in the views](https://github.com/forem/forem/blob/citizen428/drop-columns-from-user-6994/app/views/users/show.html.erb#L52) to render the User's profile name. Updating a user's name didn't reflect in my profile page locally because it's passed in as a Profile param and not a User param. Does this happen to you as well?

Otherwise I think we would need to get rid of `name` from the Basic `ProfileFieldGroup` if we want users to **always** have a name or make it "read only". I'm okay with just adding `name` to the `PROFILE_COLUMNS` so the profile page gets updated correctly, but just wanted to mention this because we had [a similar conversation here](https://github.com/forem/forem/pull/10707#issuecomment-730816702).